src/tests/EdFi.DataManagementService.Tests.E2E/Features/EqualityConstraintValidation.feature

```diff
@@ -1,13 +1,22 @@
 Feature: Equality Constraint Validation
               Equality constraints on the resource describe values that must be equal when posting a resource. An example of an equalityConstraint on bellSchedule:
     "equalityConstraints": [
-        {
-            "sourceJsonPath": "$.classPeriods[*].classPeriodReference.schoolId",
-            "targetJsonPath": "$.schoolReference.schoolId"
-        }
+    {
+    "sourceJsonPath": "$.classPeriods[*].classPeriodReference.schoolId",
+    "targetJsonPath": "$.schoolReference.schoolId"
+    }
     ]
-
         Scenario: 01 Post a valid bell schedule no equality constraint violations.
+            Given the system has these "descriptors"
+                  | descriptorname                           | codeValue         | description       | namespace                                               | shortDescription  |
+                  | educationOrganizationCategoryDescriptors | School            | School            | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | School            |
+            Given the system has these "schools"
```

Comment:
Below a GIVEN statement, you can use `and` if there are more than one condition to execute the scenario